src/services/infra/InfraService.ts

```diff
@@ -355,12 +356,17 @@ export default class InfraService {
       return errAsync(generatedDnsRecords.error)
     }
 
+    let jobStatus: SiteLaunchStatus
+    if (site.value.jobStatus === JobStatus.Ready) {
+      jobStatus = SiteLaunchStatusObject.Launched
+    } else if (site.value.jobStatus === JobStatus.Failed) {
+      jobStatus = SiteLaunchStatusObject.Failure
+    } else {
+      jobStatus = SiteLaunchStatusObject.Launching
+    }
+
```

Comment:
Two nit non-blocking comments here:

1. The variable `jobStatus` can be renamed to `siteStatus` to make the code more readable, since there are already `site.value.jobStatus` and `JobStatus`.
2. We can consider refactoring the `if-else` conditions to `switch` on `site.value.jobStatus` to make the code more extendable in case there will be more status in the future.

Reply:
sure for point 2 f0f01b1350829d8da6127c0b70424a399c9f3d58

Highly against point 1. For the purposes of our repo, our db has a `siteStatus` and a `jobStatus` column that semanitically refers to different things. I think to keep the phrasing consistent, `jobStatus` is used.

Reply:
Thanks. For point 1, I noticed that the function is returning the following:
```ts
 return okAsync<SiteLaunchDto>({
      siteStatus: jobStatus,
      dnsRecords: generatedDnsRecords.value.dnsRecords,
      siteUrl: generatedDnsRecords.value.siteUrl,
    })
```
the value of `jobStatus` is being returned as `siteStatus` property inside the `SiteLaunchDto` object eventually. So the value will be referred by `siteStatus` when the return object is used, is this not consistent with the DB column name?

Reply:
Fair point. The reason for this is because I didnt want the FE to be concerned about `SiteStatus` + `JobStatus`. ie, FE should only be concerned about whether the site is `LAUNCHED | FAILED | LAUNCHING`. 

While I am against changing the var name `jobStatus`, I am happy to change the `SiteLaunchDto` key value of `siteStatus` to reduce confusion, do you have any suggestions for the naming?

Reply:
Sure, perhaps we could change it to `siteLaunchStatus`? 
Also since the type of `jobStatus` is already `SiteLaunchStatus` in the line:
`let jobStatus: SiteLaunchStatus`
We could just change var `jobStatus` to `siteLaunchStatus`, I feel `siteLaunchStatus` is different enough from `siteStatus` to avoid any confusion. (understand the value of `siteLaunchStatus` is probably mapped to `jobStatus` in the DB column).

Reply:
sure a67518834aa2cd7312ac115e930fd88b36b2abee